azure-pipelines.yml

```diff
@@ -80,7 +80,9 @@ jobs:
       demands: assignment -equals default
     steps:
       - checkout: self
-      - bash: ci/check-changelog.sh $(fork_sha)
+      - bash: |
+          git fetch
```

Comment:
Not sure I follow. You're using `$(fork_sha)` anyway, so nothing refers to `origin/master` anywhere. I'm not sure that this will affect anything.

Can you do me a favor and explain one more time? :upside_down_face:

Reply:
You're totally right. This doesn't do anything. I've amended the PR to move the `git fetch` to the job that actually computes the fork point, where it can be more useful.

Reply:
Though, actually... I don't understand how this can be wrong on our end. I'm afraid the race condition has to have happened at the GitHub level then. We run CI on the merge commit that GitHub gives us, and find the fork point based on _its_ parents. I'm thoroughly confused now. `origin/master` never gets called here.